ci/deploy-testremote-teardown.sh

```diff
@@ -218,7 +217,6 @@ do
         exit ${retcode}
     fi
 done
-set +e
```

Comment:
The meaning of the `+` and `-` is confusing, has often confused me. Was that the problem here that you wanted to temporarily _disable_ `errexit`, but accidentally disabled it for the rest of the program?

Reply:
Exactly... n00b mistakes.

Reply:
`errexit`: sooo useful, soo powerful, soo dangerous :).